### PR TITLE
Daily PR April 4

### PR DESCRIPTION
**Pull request by pe7r**
Work with checked and it's methods. Work with styles for buttons visibility. Fixed bugs.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -24,59 +25,40 @@ class App extends Component {
...
-			this.props.checked = false
+	uncheckAll = () => {
+		const { tasks, checkedTasks } = this.state;
+		const checkedItems = [...checkedTasks]
+		tasks.forEach((item,i) => {
+			if (checkedTasks.includes(item)) {
+				checkedItems.splice(checkedItems.indexOf(item), 1)
+			}
 		})
 		this.setState({
```
Comment (ruslan-palahin) [inline src/App.js:52]: Looks like you can simply do
```
this.setState({
			checkedTasks: []
		})
```

File: src/components/ToDoItem/Forms/FormForItem.js [hunk 2]
```diff
@@ -2,23 +2,49 @@ import React from 'react'
 import '../ToDoItem.css'
 
 class FormForItem extends React.Component {
+	state = {
+		showButtons: false
+	}
+
+	handleHoverOn = () => {
```
Comment (ruslan-palahin) [inline src/components/ToDoItem/Forms/FormForItem.js]: Nice logic.
Please notice that you can do it with css:
```
.edit-delete{ display: none; }
.todo-item:hover .edit-delete{ display: block; }
```

**Review by ruslan-palahin — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (pe7r) [reply src/App.js:52]: Improved.

File: src/components/ToDoItem/Forms/FormForItem.js (hunk 2 above)
Comment (pe7r) [reply src/components/ToDoItem/Forms/FormForItem.js]: Changed.

**Review by ruslan-palahin — APPROVED** [review]

**Pull request merged**